bc_obps/reporting/migrations/00010_report_data_tables.py

```diff
@@ -0,0 +1,692 @@
+            },
+        ),
+        migrations.CreateModel(
+            name='ReportFuel',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('created_at', models.DateTimeField(auto_now_add=True, null=True)),
+                ('updated_at', models.DateTimeField(blank=True, null=True)),
+                ('archived_at', models.DateTimeField(blank=True, null=True)),
+                (
+                    'json_data',
+                    models.JSONField(db_comment='A flat JSON object representing the data collected for this model'),
+                ),
+                (
+                    'archived_by',
+                    models.ForeignKey(
+                        blank=True,
+                        null=True,
+                        on_delete=django.db.models.deletion.PROTECT,
+                        related_name='%(class)s_archived',
+                        to='registration.user',
+                    ),
+                ),
+                (
+                    'created_by',
+                    models.ForeignKey(
+                        blank=True,
+                        null=True,
+                        on_delete=django.db.models.deletion.PROTECT,
+                        related_name='%(class)s_created',
+                        to='registration.user',
+                    ),
+                ),
+                (
+                    'fuel_type',
+                    models.ForeignKey(
+                        db_comment='The fuel type this data applies to',
+                        on_delete=django.db.models.deletion.PROTECT,
+                        related_name='%(class)s_records',
+                        to='reporting.fueltype',
+                    ),
+                ),
+                (
+                    'report_version',
+                    models.ForeignKey(
+                        db_comment='The report version this data belongs to',
+                        on_delete=django.db.models.deletion.CASCADE,
+                        related_name='%(class)s_records',
+                        to='reporting.reportversion',
+                    ),
+                ),
+                (
+                    'updated_by',
+                    models.ForeignKey(
+                        blank=True,
+                        null=True,
+                        on_delete=django.db.models.deletion.PROTECT,
+                        related_name='%(class)s_updated',
+                        to='registration.user',
+                    ),
+                ),
+            ],
+            options={
+                'db_table': 'erc"."report_fuel',
+                'db_table_comment': 'A table to store the reported activity-specific data, in a JSON format',
+            },
+        ),
+        migrations.CreateModel(
```

Comment:
I don't see the fkey to the source type data object (though it is defined in the model?)

Reply:
same as earlier, it's added on Line 606